packages/components/src/guide/test/index.tsx

```diff
@@ -15,6 +15,25 @@ const defaultProps = {
 };
 
 describe( 'Guide', () => {
+	// Mock `matchMedia` so that all animations are skipped,
+	// since js-dom does not support fully CSS animations.
+	const originalMatchMedia = window.matchMedia;
+	const mockedMatchMedia = jest.fn( ( query: string ) => {
+		if ( /prefers-reduced-motion/.test( query ) ) {
+			return { matches: true } as ReturnType< typeof window.matchMedia >;
+		}
+
+		return originalMatchMedia( query );
+	} );
+
+	beforeAll( () => {
+		window.matchMedia = jest.fn( mockedMatchMedia );
+	} );
+
+	afterAll( () => {
+		window.matchMedia = originalMatchMedia;
+	} );
+
```

Comment:
The `matchMedia` mock is getting repetitive. We might want to add a helper to allow reusability.

Reply:
I'm not sure we have repo-wise helpers — but I can definitely add a utility at least for the components package.

Reply:
Pushed https://github.com/WordPress/gutenberg/pull/65203/commits/ba25974eb80b825c90ae362bd57064d4054fc5e7 — let me know if you'd extract if in a different way, or if you think there's a better way to share those mocks around

Reply:
Since this helper will be necessary any time we need to use `window.matchMedia`, let's add it as a global mock.

Will push a commit to show what I mean.

Reply:
See a7aeb6ce4ee69997fc26ac4442f88e68f8eeb10d. Here's what we're doing:

- Creating the mock. Setting it up directly, globally.
- Including the mock in `setupFilesAfterEnv` because we need to run it later, after the environment has been setup, due to the fact it requires `beforeAll` / `afterAll()` access.
- Removing the previous mock
- Removing the manual import and setup in each component test suite

Let me know what you think!

Reply:
Hmm, this actually breaks some other tests, let me take a look.

Reply:
Extracted to #65438

Reply:
Thanks. Now that it landed, the branch will need a rebase